### PR TITLE
Synchronize with latest

### PR DESCRIPTION
**Pull request by timbotnik**
@zhenyasav - this is an attempt to bring all of these packages up-to-date with our internal codebase.  We will need to re-publish these again to atmosphere once merged.

### CODE REVIEW

File: packages/code-block/CodeBlock.jsx [hunk 1]
```diff
@@ -24,12 +25,39 @@ if (Chromatic) {
...
     specs: [
       new Chromatic.Spec('default', {props: {
-        code: `{ "json": "value" };
-        const square = function(x) { return Math.pow(x, 2); }
-        // comment`
+        code: `{
+          "galaxy.meteor.com": {
+            "env": {
+              "ROOT_URL": "https://atmospherejs.com",
+              "MONGO_URL": "mongodb://atmosphere:asdf-asdf@candidate.6.mongolayer.com:10396,candidate.5.mongolayer.com:10513/atmosphere?replicaSet=set-55b0177a159f072bc8001267",
```
Comment (zhenyasav) [inline packages/code-block/CodeBlock.jsx]: are these real ? rotated?

File: packages/utils/utils.js [hunk 2]
```diff
@@ -80,6 +70,19 @@ Utils = _.extend(Utils || {}, {
       .replace(/^-/, '')
       .replace(/\/-/g, '/');
   },
+  toSplitWords(string) {
+    return this.capitalize(string.replace(/([A-Z])/g, (c) => ` ${c}`));
+  },
+  // User-visible form of versionId
+  formatVersionId(versionId) {
```
Comment (zhenyasav) [inline packages/utils/utils.js]: gxy specific?

File: packages/utils/utils.js [hunk 3]
```diff
@@ -80,6 +70,19 @@ Utils = _.extend(Utils || {}, {
...
+    // if the version id is "old" (i.e doesn't contain a version number, this is the best we can do)
+    return versionId.split('-')[1] || '-';
+  },
+  formatContainerId(containerId) {
```
Comment (zhenyasav) [inline packages/utils/utils.js]: gxy specific?

**Comment by zhenyasav** [comment]
:shipit:

File: packages/utils/utils.js (hunk 3 above)
Comment (timbotnik) [inline packages/utils/utils.js]: Yeah let's move that to AppVersions.helpers

**Pull request merged**